libs/shared/src/packages/table-helpers/src/lib/table/TabTable/TabTable.tsx

```diff
@@ -12,6 +12,7 @@ type TabTableProps<T> = {
   error: Error | null;
   resourceName: string;
   height?: number;
+  additionalGridOptions?: GridOptions;

```

Comment:
I see this prop was added but I cant find it being used anywhere. Was this intentional?

Reply:
In this case i did not use it because I moved the changes to the default grid options, but i thought it would be fine to leave in as it will probably be useful later.